### ISSUE TITLE
What is Companion Server?

### ISSUE THREAD

**Issue by dariusapostol** [ytmdesktop/ytmdesktop#74]
I just found the new setting and I don't know how it works... 😂

**Comment by MrMelon54** [comment]
I think its an unfinished feature for remotely changing song. But it doesn't appear to be finished or it required a mobile app that I can't find.

**Comment by diegocade1** [comment]
> 
> 
> I think its an unfinished feature for remotely changing song. But it doesn't appear to be finished or it required a mobile app that I can't find.

I just thought the same, hopefully they will explain how to use this later in the readme file

**Comment by adlerluiz** [comment]
It's an application I'm developing but still unfinished

![image](https://user-images.githubusercontent.com/2112638/71706248-7c44a400-2dc2-11ea-91ca-8166cb955fc2.png)

**Comment by MrMelon54** [comment]
Ah a mobile app to control the pc client. That makes total sense 👍🏻

**Comment by dadi11** [comment]
any progress with that app? it's a game changer for me :D if it's still not finished, maybe you can recommend some 3rd party one?

**Comment by lucas-luchack** [comment]
I wish to know if you can upload the .apk for a "beta review"?
If you can't, do you have a release date?

**Comment by adlerluiz** [comment]
Hello, I already sent the app to google play, I'm waiting for them to review, I already have the link but it's not available yet

https://play.google.com/store/apps/details?id=app.ytmdesktop.remote
![image](https://user-images.githubusercontent.com/2112638/87452234-5866e400-c5d7-11ea-994b-78ca44442b2c.png)

prints
![1](https://user-images.githubusercontent.com/2112638/87452329-7af8fd00-c5d7-11ea-97ef-18db2896f0d1.png)
![2](https://user-images.githubusercontent.com/2112638/87452333-7c2a2a00-c5d7-11ea-9f4d-72e46c64741a.png)
![3](https://user-images.githubusercontent.com/2112638/87452335-7cc2c080-c5d7-11ea-8f24-1e6c34173785.png)
![4](https://user-images.githubusercontent.com/2112638/87452336-7d5b5700-c5d7-11ea-920e-9d3d1a739ced.png)

**Comment by dariusapostol** [comment]
Thanks a lot! I will play around with it and give you feedback as soon as possible!
-------- Original message --------From: Adler Luiz <notifications@github.com> Date: 7/14/20  19:40  (GMT+02:00) To: ytmdesktop/ytmdesktop <ytmdesktop@noreply.github.com> Cc: dariusapostol <darius_apostol29@yahoo.com>, State change <state_change@noreply.github.com> Subject: Re: [ytmdesktop/ytmdesktop] What is Companion Server? (#74) 
Hello, I already sent the app to google play, I'm waiting for them to review, I already have the link but it's not available yet
https://play.google.com/store/apps/details?id=app.ytmdesktop.remote

prints

**Comment by lucas-luchack** [comment]
Hello!
First feedback from me and I couldn't test very far beacause the application firmly refuses to connect to my computer.

I activated the server on the desktop app, I scanned the QR code with the phone app but the connection is not etablished.

**Information about my configuration**:
- PC wired to the internet
- Xiaomi Mi 8 Lite
- MIUI Global 11.0.2
- Android 10 QKQ1.190910.002
- Security fix of 2020-04-01
- No password for the companion server

**Screen**:
![IMG_20200715_115429.jpg](https://user-images.githubusercontent.com/39491773/87532695-d5579380-c693-11ea-89c5-698ebcc76764.jpg)
![IMG_20200715_115454.jpg](https://user-images.githubusercontent.com/39491773/87532712-d983b100-c693-11ea-882b-9bc7ad8b2c9f.jpg)
![IMG_20200715_115928.jpg](https://user-images.githubusercontent.com/39491773/87532720-dc7ea180-c693-11ea-985a-c3159636050e.jpg)
